packages/flame/lib/src/components/mixins/tappable.dart

```diff
@@ -61,6 +61,7 @@ mixin Tappable on Component {
   }
 }
 
+// TODO(spydon): these methods should return the result of propagateToChildren.
```

Comment:
Consider opening an issue for improved visibility

Reply:
Opened #1211